linux-create-vm.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+#install git
+echo "git should be installed"
+if ( which git )
+then 
+  echo "git already installed"
+else 
+  echo "installing git"
+  sudo apt install git
+fi
+
+#install nano
+echo "nano should be installed"
+if ( which nano )
+then
+  echo "nano already installed"
+else
+  echo "installing nano"
+  sudo apt install nano 
+fi
+
+#install snap
+echo "nano should be installed"
```

Comment:
I am not sure why we are leading off with an echo, although It probably works this way. You should get the spacing right, and this particular code block is snap. Yet it says nano should be installed.